compiler/rustc_errors/src/lib.rs

```diff
@@ -973,6 +973,7 @@ impl Handler {
         self.inner.borrow_mut().span_bug(span, msg)
     }
 
+    /// For documentation on this, see `Session::delay_span_bug`.
```

Comment:
Can you make this an intra doc link? (and below please)

Reply:
session is in a downstream crate, so that doesn't work